sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/filter/And.java

```diff
@@ -20,20 +20,18 @@
 import org.apache.spark.annotation.Evolving;
 
 /**
- * A filter that evaluates to {@code true} iff both {@code left} and {@code right} evaluate to
+ * A predicate that evaluates to {@code true} if both {@code left} and {@code right} evaluate to
```

Comment:
`iff` means if and only if, which is correct here. We shouldn't change it.

Reply:
same to other places.

Reply:
Thanks for the explanation.